src/services/__mocks__/usersService.ts

```diff
@@ -47,7 +47,9 @@ export const mockedUsers: Array<PartyUserDetail> = [
       },
     ],
     isCurrentUser: false,
-    certification: true,
+    // certifiedName: true,
+    // certifiedSurname: true,
+    // certifiedMail: true,
   },
```

Comment:
keep it clean (also on the other lines)
```suggestion
  },
```